### ISSUE TITLE
[2.0] First MaterialColumn on Masonry is not appearing

### ISSUE THREAD

**Issue by richelivargas** [GwtMaterialDesign/gwt-material-addins#168]
Example, I am adding 10 MaterialColumn itens, but only 9 are appearing on Masonry.

**Comment by kevzlou7979** [comment]
Hi there, it seems that this is not an issue, I tried to reproduce it with the same demo showcase but I can't see any issue. Can you please try to submit your code. It might be some logic issue :)

**Comment by richelivargas** [comment]
My test:

```
masonry.clear(); // At moment masonry is empty
masonry.add(materialColumn1); // Not appearing
masonry.add(materialColumn2); // It's Ok
masonry.add(materialColumn3); // It's Ok
masonry.add(materialColumn4); // It's Ok
masonry.clear();
masonry.add(materialColumn1);  // Not appearing
masonry.add(materialColumn2);  // Not appearing
masonry.add(materialColumn3);  // Not appearing
masonry.add(materialColumn4);  // Not appearing
```

**Comment by steeveb972** [comment]
I don't know if this is linked or not but there is a defect in MaterialMasonry. Calling masonry.clear() remove all children from masonry even the sizer div that is set up in the constructor. I think that all update methods on children widgets have to be checked in this widget. As a workaround, the idea is to clean all children but first.

**Comment by kevzlou7979** [comment]
I just checked this, the problem is on ``` clear() ``` method. We need to explicitly call ``` super.remove(widget) ``` except the ``` sizerDiv ```. I might submit a fixed this coming 2.1 release.
